internal/providers/amazon/eks/workflow/create_eks_control_plane_activity.go

```diff
@@ -215,6 +216,10 @@ func waitUntilClusterCreateCompleteWithContext(eksSvc *eks.EKS, ctx aws.Context,
 		},
 		Logger: eksSvc.Config.Logger,
 		NewRequest: func(opts []request.Option) (*request.Request, error) {
+
+			count = count + 1
```

Comment:
`count++` or `count += 1` would be simpler

Reply:
fixed